react-components/src/components/RevealContainer/RevealContainer.tsx

```diff
@@ -7,10 +7,13 @@ import { Cognite3DViewer, type Cognite3DViewerOptions } from '@cognite/reveal';
 import { RevealContext } from './RevealContext';
 import { type Color } from 'three';
 import { ModelsLoadingStateContext } from '../Reveal3DResources/ModelsLoadingContext';
+import { SDKProvider } from './SDKProvider';
+import { QueryClientProvider, QueryClient } from '@tanstack/react-query';
 
 type RevealContainerProps = {
   color?: Color;
   sdk: CogniteClient;
```

Comment:
We should be using `sdk` from `@cognite/sdk-provider` than passing it as props, not relevant to this PR 😃

Reply:
We discussed it with Christopher, he said that we don't want to use that package because it's too simple to add the code ourselves. I've copied the code from there, so should be no problem.